### ISSUE TITLE
Import/export feature does not show up soon the cart page

### ISSUE THREAD

**Issue by plabadie** [SAP/spartacus#14281]
Issue from branch `feature/GH-14172` on PR #14246

On the cart page, we get this error, which is likely related:

```
core.js:6456 ERROR TypeError: Cannot read properties of undefined (reading '__source')
```

**Comment by WeizhengSap** [comment]
This fix depends on #14276

**Comment by WeizhengSap** [comment]
close it since it is fixed in #14312
